sdk/aqueduct/integrations/dynamic_k8s_integration.py

```diff
@@ -24,7 +42,9 @@ def status(self) -> str:
 
         return engine_statuses[self._metadata.name].status.value
 
-    def create(self) -> None:
+    def create(self, config_delta: Union[Dict[str, str], DynamicK8sConfig] = {}) -> None:
```

Comment:
These are user facing functions so we definitely need to document what `config_delta` means and does. I think I prefer the term `config` too. It's not really a delta if we're updating it to the new end state right?

Reply:
I added explanation of what `config_delta` means. It is a "delta" so I'd partial to keeping this name.